calico_containers/calicoctl.py

```diff
@@ -1217,6 +1342,12 @@ def print_paragraph(msg):
                                   arguments["--interface"])
                 if arguments["remove"]:
                     container_remove(arguments["<CONTAINER>"])
+        elif arguments["bgp-node-mesh"]:
+            enable = arguments["on"]
+            bgp_node_mesh(TRI_STATE_ON_OFF_MAP[(bool(arguments["off"]),
```

Comment:
Clever, but it's hard to follow what this is doing.  Why not just

```
if arguments["on"] or arguments["off"]:
    set_bgp_node_mesh(arguments["on"])
else:
    show_bgp_node_mesh()
```

Right now you're doing this mapping to `True`, `False` and `None`, and then as soon as you get into the function you check if it's `None` and do one, and if not do a different thing.